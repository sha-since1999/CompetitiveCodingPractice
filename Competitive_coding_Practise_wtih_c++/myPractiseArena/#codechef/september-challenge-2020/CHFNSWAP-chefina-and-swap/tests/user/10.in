7
304956304
095374069
073945763
983457603
1123452
938475693
947520456