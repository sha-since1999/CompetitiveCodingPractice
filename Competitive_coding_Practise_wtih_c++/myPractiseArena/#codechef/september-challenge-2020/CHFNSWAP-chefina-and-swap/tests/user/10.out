89319634
0
21658213
288048064
329052
0
277522317
